docs/api/checkouts.rst

```diff
@@ -785,7 +770,262 @@ Final response contains the full order information, now you can call the purchas
 
 .. code-block:: json
 
-    {"to": "do"}
+    {
+        "adjustments": [
+            {
+                "amount": 0,
+                "created_at": "2014-12-15T13:37:29+0000",
+                "description": "No tax (0%)",
+                "id": 205,
+                "type": "tax",
+                "locked": false,
+                "neutral": false,
+                "updated_at": "2014-12-15T13:37:29+0000"
+            },
+            {
+                "amount": 5000,
+                "created_at": "2014-12-15T14:30:41+0000",
+                "description": "FedEx World Shipping",
+                "id": 207,
+                "type": "shipping",
+                "locked": false,
+                "neutral": false,
+                "updated_at": "2014-12-15T14:30:41+0000"
+            },
+            {
+                "amount": -250,
+                "created_at": "2014-12-15T14:30:41+0000",
+                "description": "Christmas Sale for orders over 100 EUR.",
+                "id": 208,
+                "type": "promotion",
+                "locked": false,
+                "neutral": false,
+                "updated_at": "2014-12-15T14:30:41+0000"
+            }
+        ],
+        "adjustments_total": 4750,
+        "billing_address": {
+            "_links": {
+                "country": {
+                    "href": "/app_dev.php/api/countries/9"
+                }
+            },
+            "city": "New York",
+            "created_at": "2014-12-15T13:37:28+0000",
+            "first_name": "John",
+            "id": 106,
+            "last_name": "Doe",
+            "postcode": "12435",
+            "street": "Test",
+            "updated_at": "2014-12-15T13:37:29+0000"
```

Comment:
I would keep the same order as in model:
1. id
2. firstName
3. lastName
4. where is the *countryCode*?
5. street
6. city
7. postcode
and then
8. createdAt
9. updatedAt